embedded-tests/src/test/java/org/apache/druid/testing/embedded/docker/CustomNodeRoleDockerTest.java

```diff
@@ -89,5 +89,17 @@ public Map<String, String> getDefaultProperties()
     {
       return Map.of();
     }
+
+    @Override
+    public Integer getExposedOperatorPort()
+    {
+      return 9302;
+    }
+
+    @Override
+    public Map<String, String> getOperatorConfiguration()
+    {
+      return Map.of();
+    }
```

Comment:
Should not be needed anymore.

Reply:
Yeah I am pushing the final changes, earlier commit was WIP